src/redux/slices/uv-debug-texture-slice.ts

```diff
@@ -25,16 +25,21 @@ const uvDebugTextureSlice = createSlice({
       fetchUVDebugTexture.fulfilled,
       (
         state: uvDebugTextureState,
-        action: PayloadAction<{ imageUrl: string; image: ImageBitmap | null }>
+        action: PayloadAction<{
+          imageUrl: string;
+          image: ImageBitmap | null;
```

Comment:
Does it make sense to push `null` to `image`?

Reply:
Removed null from the type.